lib/rules-fr.js

```diff
@@ -0,0 +1,105 @@
+// A set of predefined rules that are especially applicable to French strings.
+
+const en = require('./rules-en');
+const { RemoveRule, FunctionRule } = require('./rules');
+const normalize = require('./normalize');
+
+const COMMON_PUNCTUATION_STR = en.COMMON_PUNCTUATION_STR + `«»`;
+const COMMON_SYMBOLS_STR = en.COMMON_SYMBOLS_STR + `€`;
+const COMMON_PUNCTUATION_AND_SYMBOLS_STR =
+  COMMON_PUNCTUATION_STR + COMMON_SYMBOLS_STR;
+const COMMON_PUNCTUATION_AND_SYMBOLS = new RemoveRule(
+  COMMON_PUNCTUATION_AND_SYMBOLS_STR,
+  {
+    name: 'common punctuation and symbols',
+  }
+);
+
+const replaceLigatures = str => {
+  return str
+    .replace(/æ/g, 'ae')
+    .replace(/Æ/g, 'AE')
+    .replace(/œ/g, 'oe')
+    .replace(/Œ/g, 'OE');
+};
+
+const LIGATURES = new FunctionRule(
+  (s1, s2) => {
+    return [replaceLigatures(s1), replaceLigatures(s2)];
+  },
+  { name: 'ligatures replaced' }
+);
+
+const DECOMPOSED_ACCENTS = ['́', '̀'];
+
+const isAccentedCharacter = char => {
+  const normalized = normalize(char, 'NFD').split('');
+  return normalized.slice(1).some(char => DECOMPOSED_ACCENTS.includes(char));
+};
```

Comment:
[String.prototype.normalize()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/normalize) documentation is super helpful here

Reply:
Yes -- there is some similar stuff going on around [here](https://github.com/sandinmyjoints/equivalency/blob/CMCAR-29-equivalency-add-support-for-french-equivalency/lib/rules.js#L213), but not quite the same as what you're doing.